### PR TITLE
1 02 11 logical operators

### PR DESCRIPTION
**Pull request by Zim123**
Pull request for translation https://github.com/javascript-tutorial/uk.javascript.info/issues/29

### CODE REVIEW

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-Although they are called "logical", they can be applied to values of any type, not only boolean. Their result can also be of any type.
+Хоча вони називаються "логічними", вони можуть бути застосовані до значень будь-якого типу, не тільки булевих. Їх результати також можуть бути будь-якого типу.
 
-Let's see the details.
+Подивимось більш детально.
 
 ## || (OR)
 
-The "OR" operator is represented with two vertical line symbols:
+Оператор "OR" представлений двома символами вертикальної лінії:
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/11-logical-operators/article.md]: Оператор "АБО"

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 2]
```diff
@@ -27,113 +27,113 @@ alert( true || false );  // true
...
 
-Given multiple OR'ed values:
+Дано кілька значень, розділених опертором OR:
 
 ```js
 result = value1 || value2 || value3;
 ```
 
-The OR `||` operator does the following:
+Оператор OR `||` робить наступне:
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/11-logical-operators/article.md]: Оператор **АБО** `||`

File: 1-js/02-first-steps/11-logical-operators/article.md [hunk 3]
```diff
@@ -27,113 +27,113 @@ alert( true || false );  // true
...
-- If all operands have been evaluated (i.e. all were `false`), returns the last operand.
+- Обчислює операнди зліва направо.
+- Перетворює значення кожного операнда на булеве. Якщо результат `true`, зупиняється і повертає початкове значення цього операнда.
+- Якщо всі операнди були обчисленні (тобто усі були `false`), повертає останній операнд.
 
-A value is returned in its original form, without the conversion.
+Значення повертається у первісному вигляді без конвертації.
 
-In other words, a chain of OR `"||"` returns the first truthy value or the last one if no truthy value is found.
+Іншими словами, ланцюжок с OR `"||"` повертає перше правдиве значення або останнє, якщо не знайдено правдивого значення.
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/11-logical-operators/article.md]: ланцюжок з АБО `"||"`

**Comment by tarasyyyk** [comment]
@Zim123 thanks for your contribution!

@all-contributors add @Zim123 for translation.

**Comment by Zim123** [comment]
@tarasyyyk no problem :)
I'm having some changes related to work, so I hope it will stabilize soon, and I will be back to proceed with the translation :)
